crates/spidermonkey-embedding-splicer/src/bindgen.rs

```diff
@@ -104,6 +106,9 @@ struct JsBindgen<'a> {
     resource_directions: HashMap<TypeId, AbiVariant>,
 
     imported_resources: BTreeSet<TypeId>,
+
+    /// Features that were enabled at the time of generation
+    features: &'a Vec<Features>,
```

Comment:
I think `Features` is a bit unfortunate name as this implies it is already a list, but that's generated right?